ament_index_python/test/test_ament_index_python.py

```diff
@@ -181,4 +196,110 @@ def get_package_share_directory_test(package_name, expect_prefix):
     except PackageNotFoundError:
         pass
     except Exception as exc:
-        assert False, 'Expected PackageNotFoundError, got: {}'.format(type(exc))
+        assert False, 'Expected PackageNotFoundError, got: {}' \
+            .format(type(exc))
+
+
+def test_get_resource_types():
+    # Empty ament prefix will raise env error
+    set_ament_prefix_path([''])
+    try:
+        get_resource_types()
+    except EnvironmentError:
+        pass
+    except Exception as exc:
+        assert False, 'Expected EnvironmentError, got: {}'.format(type(exc))
+
+    set_ament_prefix_path(['prefix1', 'prefix2'])
+    resources = get_resource_types()
+    assert resources == {'resource_type1',
+                         'resource_type2',
+                         'resource_type3',
+                         'resource_type4',
+                         'resource_type5',
+                         'packages'}, 'Expected resources to ' + \
+        'be found: resource_type1,  resource_type2, ' + \
+        'resource_type3,  resource_type4, ' + \
+        'resource_type5 and packages'
```

Comment:
You don't need `+` for joining strings literals, `python` automatically joins them without it.
If you need to escape the new line repeatedly, prefer using brackets.
Finally, accordding to the style guide, don't over ident to align (after a new line, you can't ident more than once).
nit: `resources to be found` -> `resources to be`
```suggestion
    assert resources == {
        'resource_type1',
        'resource_type2',
        'resource_type3',
        'resource_type4',
        'resource_type5',
        'packages'
    }, ('Expected resources to be : resource_type1,  resource_type2, '
        'resource_type3,  resource_type4, resource_type5 and packages')
```

Reply:
Same comment about style elsewhere.